tests/scripts/k8s-vagrant-multi-node.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
was the `-x` flag just for debugging while you were working on the script, or do you want it in the final product?